mlflow/openai/__init__.py

```diff
@@ -346,11 +346,16 @@ def save_model(
     _validate_and_prepare_target_save_path(path)
     code_dir_subpath = _validate_and_copy_code_paths(code_paths, path)
     task = _get_task_name(task)
+    _get_api_config()
```

Comment:
@santiagxf Why do we need to call `_get_api_config` here?

Reply:
> @santiagxf Why do we need to call `_get_api_config` here?

Reply:
Great catch. I must have had it on the clipboard and pasted it accidentally. Indeed it makes no sense.